AFQ/api.py

```diff
@@ -157,7 +157,7 @@ def __init__(self,
                  dask_it=False,
                  force_recompute=False,
                  scalars=["dti_fa", "dti_md"],
-                 wm_criterion=0.1,
+                 wm_criterion={"dti_fa": ">0.1", "dti_md": "<0.003"},
```

Comment:
Usually, when I find myself trying to write code in strings, to be parsed by something else, it means that something is overly (or underly) complicated. 

I think that there are two (possibly) more robust ways to do this: 

1. Expand the dictionary a bit to have: `{'lb':{'dti_fa': 0.1}, 'ub': {'dti_md':0.0003}}`. One advantage of that is that you can choose ranges. 

2. Use two key-word arguments: 

```
wm_criterion_lb={'dti_fa':0.1}
wm_criterion_ub={'dti_md':0.003} 
```

Reply:
I agree that code in strings should be avoided. I like the first option because it keeps everything in the same argument, which could also be a list if you want to use labels.